GooglePlayInstant/Editor/PlayInstantQuickDeployWindow.cs

```diff
@@ -197,10 +200,29 @@ private void OnGuiCreateBuildSelect()
             EditorGUILayout.Space();
             EditorGUILayout.BeginHorizontal();
             EditorGUILayout.LabelField("APK File Name", GUILayout.MinWidth(FieldMinWidth));
-            EditorGUILayout.TextField("c:\\base.apk", GUILayout.MinWidth(FieldMinWidth));
+            Il2cppBuilder.ApkPathName =
+                EditorGUILayout.TextField(Il2cppBuilder.ApkPathName, GUILayout.MinWidth(FieldMinWidth));
+            if (GUILayout.Button("Browse", GUILayout.Width(ShortButtonWidth)))
+            {
+                Il2cppBuilder.ApkPathName = EditorUtility.SaveFilePanel("Choose file name and location",
+                    Path.GetDirectoryName(Il2cppBuilder.ApkPathName), Path.GetFileName(Il2cppBuilder.ApkPathName),
+                    "apk");
+            }
+
             EditorGUILayout.EndHorizontal();
             EditorGUILayout.Space();
-            GUILayout.Button("Build Base APK", GUILayout.Width(ButtonWidth));
+
+            if (!Il2cppBuilder.ProjectIsUsingIl2cpp)
+            {
+                EditorGUILayout.LabelField(
+                    "Warning: You are not using IL2CPP as the project's scripting backend, and this may result into a larger apk size than necessary.",
+                    EditorStyles.wordWrappedLabel);
+            }
+
+            if (GUILayout.Button("Build Base APK", GUILayout.Width(ButtonWidth)))
```

Comment:
Another option instead of having the IL2CPP warning inline and asking them to change it is to ask a question when they click build but before we build:

"Your project is using the Mono scripting runtime. Would you like to build the APK with IL2CPP and Engine Stripping to improve game performance and reduce APK size?"  Yes / No.

Reply:
Resolved.

Used suggested approach.